lib/runTracker.js

```diff
@@ -10,10 +10,6 @@ function runTracker (opts, tracker, cb) {
 
   run(opts, tracker, cb)
 
-  process.once('SIGINT', () => {
```

Comment:
This was the cause for mem leak warning in the console.
It wasn't a use-case issue though. It was coming from wasm.test.js where we're running LOTS of parallel tests and each one was binding this SIGINT event.